syntax = "proto3";
package RPC;

service TemperatureService {
    rpc StreamTemperature (StreamRequest) returns (stream TemperatureData);
}

message StreamRequest {
}

message TemperatureData {
    float temperature = 1;
    string timestamp = 2;
}